pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -142,7 +138,21 @@ func (Plugin) OnInboundRouteConfiguration(in *plugin.InputParams, routeConfigura
 	default:
 		log.Warn("Unknown listener type in mixer#OnOutboundRouteConfiguration")
 	}
+}
+
+func buildMixerOpaqueConfig(disableCheck, _ /*disableForward*/ bool, destinationService string) *mccpb.ServiceConfig {
+	out := &mccpb.ServiceConfig{
+		// Report calls are never disabled. Disable forward is currently not in the proto.
+		DisableCheckCalls: disableCheck,
+	}
+	if destinationService != "" {
+		out.MixerAttributes = &mpb.Attributes{}
+		out.MixerAttributes.Attributes = map[string]*mpb.Attributes_AttributeValue{
+			v1.AttrDestinationService: {Value: &mpb.Attributes_AttributeValue_StringValue{StringValue: destinationService}},
+		}
+	}
```

Comment:
We should have more service config than just adding a "destination.service" attributes.

Reply:
Let's discuss in a followup. This gives us the basics at least.